lib/UtapiClient.js

```diff
@@ -264,6 +275,40 @@ class UtapiClient {
     }
 
     /**
+    * Publish an event in the bucket's dedicated channel to notify subscribers
+    * for performed actions with side effects
+    * @param {string} metric - metric to be published
+    * @param {object} params - params for the metrics
+    * @param {string} [params.bucket] - (optional) bucket name
+    * @param {array} [params.objects] - (optional) objects affected
+    * @param {object} log - logger
+    * @param {callback} [callback] - (optional) callback to call
+    * @return {undefined}
+    */
+    _publishEvent(metric, params, log, callback) {
+        const { bucket, objects } = params;
+        let message = metric;
+        if (Array.isArray(objects)) {
+            objects.forEach(object => {
+                message = `${message}:${object}`;
```

Comment:
Do you mean to append here?

Reply:
yes

Reply:
For readability, I recommend
```js
const message = keys.reduce((a, b) => `{a}:${b}`, metric);
```
Note, I renamed objects to keys.